lib/galaxy/tools/synchronize_two_collections_list.xml

```diff
@@ -0,0 +1,196 @@
+<tool id="__SYNCHRONIZELISTS__"
```

Comment:
Can you add this to https://github.com/galaxyproject/galaxy/blob/dev/test/functional/tools/sample_tool_conf.xml#L294 and https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/tool_conf.xml.sample#L45 ?

With that you can should be able to run the tool test with `./run_tests.sh -framework -id __SYNCHRONIZELISTS__`